netstats/src/views/layout.jade

```diff
@@ -3,34 +3,55 @@ doctype html
 html(ng-app="netStatsApp")
   head
     meta(name="viewport", content="width=device-width, initial-scale=1.0, maximum-scale=1.0")
-    title Kowala Test Network
+    title Kowala - kUSD Test Network
     style(type="text/css") [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak { display: none !important; }
-    link(rel='stylesheet', href='//fonts.googleapis.com/css?family=Montserrat:200,300,400,500,600"')
-    link(rel='stylesheet', href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css')
+    link(rel="stylesheet", href="https://fonts.googleapis.com/css?family=Poppins:300,400,500")
+    link(rel="stylesheet", href="https://cdn.kowala.tech/styles/kowala.css")
     link(rel='stylesheet', href='/css/netstats.min.css')
-  body
-    section#nav.container-fluid
-      nav.navbar(ng-controller='NavCtrl')
-        .navbar-header
-          button.navbar-toggle(type='button', data-toggle='collapse', data-target='#Navbar')
-            i.fa.fa-bars
-          a.navbar-brand(href="/")
-            img(src="/kowala-zygote-testnet.svg" height="25")
-        #Navbar.collapse.navbar-collapse
-          ul.navbar-nav.nav.navbar-right
-            li
-              a(href="/stats") Network stats
-            li
-              a(href="https://explorer.zygote.kowala.tech", target="_blank") BlockChain Explorer
-            li
-              a(href="https://faucet.zygote.kowala.tech", target="_blank") Coin faucet
-            li
-              a(href="https://github.com/kowala-tech/kcoin", target="_blank") Github
-            li
-              a(href="https://docs.kowala.tech", target="_blank") Documentation
-            li
-              a(href="https://t.me/kowalaofficial", target="_blank") Telegram Community
-    section#content.container-fluid
-      block content
+  body.light-blue-background.light-#fff-text
+    #mobile-menu(data-reveal="")
+      img(src="https://cdn.kowala.tech/images/kowala/kowala-white.svg" title="Kowala")
+      ul
+        li
+          a(href="https://explorer.zygote.kowala.tech" rel="noopener" target="_blank") BLOCK EXPLORER
```

Comment:
There will soon be prod versions of these links. Can we link dev -> dev and prod -> prod?

Reply:
Updated and added social media meta